### ISSUE TITLE
Screen reader do not read out the current slide image description

### ISSUE THREAD

**Issue by carinafu** [nolimits4web/swiper#4536]
- Swiper Version: EXACT RELEASE VERSION OR COMMIT HASH, HERE.
- Platform/Target and Browser Versions: PLATFORM CLIENT YOU ARE TARGETING SUCH AS macOS, Windows, CORDOVA, IOS, ANDROID, CHROME, ETC.

## What You Did

EXPLAIN WHAT YOU DID, PREFERABLY WITH CODE EXAMPLES, HERE.

 https://stackblitz.com/...

## Expected Behavior

I create a demo with swiper angular.
the actual slide content is images with alt description.
when I use voiceover, navigate to [prev] and [next] button, it suppose to read out the current image description.

## Actual Behavior
when I use voiceover, navigate to [prev] and [next] button, it does not read out the current image description.
I can see the aria-live, aria-controls attributes are all there correctly. But no aria-hidden for all the slides. not sure if it is the case. but I add the aria-hidden attribute on slidechange event, it still not work.

**Comment by vltansky** [comment]
Hey, can you reproduce the bug on the "Core" version?
https://swiperjs.com/demos/020-navigation/core.html

**Comment by vltansky** [comment]
Closed due to authors inactivity.
